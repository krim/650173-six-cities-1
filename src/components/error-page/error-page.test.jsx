import React from 'react';
import renderer from 'react-test-renderer';

import ErrorPage from './error-page';

describe(`ErrorPage`, () => {
  it(`renders component correctly`, () => {
    const tree = renderer.create(
        <ErrorPage />
    ).
    toJSON();

    expect(tree).toMatchSnapshot();
  });
});
