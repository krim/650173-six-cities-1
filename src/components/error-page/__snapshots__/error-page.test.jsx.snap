// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage renders component correctly 1`] = `
<div
  style={
    Object {
      "color": "#FF0000",
      "fontSize": "2em",
      "marginTop": "10em",
      "textAlign": "center",
    }
  }
>
  Something goes wrong. Please try again later.
  <br />
  <a
    href="/"
    style={
      Object {
        "color": "#000000",
        "textDecoration": "underline",
      }
    }
  >
    Return to the main page
  </a>
</div>
`;
